azure-functions-maven-plugin/src/main/java/com/microsoft/azure/maven/function/DeployMojo.java

```diff
@@ -37,12 +37,14 @@
 public class DeployMojo extends AbstractFunctionMojo {
     public static final String FUNCTION_DEPLOY_START = "Starting deployment to Azure Function App ";
     public static final String FUNCTION_DEPLOY_SUCCESS =
-            "Successfully deployed Azure Function App at https://%s.azurewebsites.net";
+        "Successfully deployed Azure Function App at https://%s.azurewebsites.net";
```

Comment:
"Azure Functions app"

Reply:
The name of the service is "Azure Functions", Please fix all.

Reply:
I will fix them all in next PR

Reply:
I prefer using `Azure Function App` here.

Azure Functions is the service name. But for a function instance, the name is `Function App`.

See:

![image](https://user-images.githubusercontent.com/6193897/43629494-ff86ed6c-972f-11e8-92fe-87b3dbabf8bc.png)

Reply:
Good to know that they have an official term `function app`. In this case, we don't need to mention Azure brand every time. So I would suggest `function app` and the message looks like:
Successfully deployed the function app at ...